helper.php

```diff
@@ -42,10 +42,20 @@ public function getDBHelper() {
 
     public function add($page) {
         $sqlite = $this->getDBHelper();
+        if(!$sqlite) return;
 
-        $sql = "INSERT OR REPLACE INTO toppages (page, value)
-                  VALUES ( ?, COALESCE( (SELECT value FROM toppages WHERE page = ?) + 1, 1))";
-        $res = $sqlite->query($sql, $page, $page);
+        try {
+            $translation = new helper_plugin_translation();
```

Comment:
use plugin_load() to load the translation plugin. It will return null if the plugin is not available.